youtube_dl/postprocessor/ffmpeg.py

```diff
@@ -451,7 +451,7 @@ def add(meta_list, info_list=None):
         # 4. http://atomicparsley.sourceforge.net/mpeg-4files.html
```

Comment:
The `add()` function above could be improved based on yt-dlp (import `T()`, `traverse_obj()`, `txt_or_none()`, `variadic()` from `..utils`):
```py
        def add(meta_list, info_list=None):
            if not meta_list:
                return
            meta_list = variadic(meta_list)
            info_list = variadic(info_list) if info_list else meta_list
            value = traverse_obj(info, (info_list, T(txt_or_none), any))
            if value not in (None, ''):
                value = value.replace('\0', '') # nul character cannot be passed in command line
                metadata.update((meta_f, value) for meta_f in meta_list)
```